react/components/SKUSelector/components/SelectorItem.tsx

```diff
@@ -40,22 +42,37 @@ const SelectorItem: FC<Props> = ({
   imageUrl,
   imageLabel,
   isImpossible,
+  imageHeight,
+  imageWidth,
 }) => {
   const discount = getDiscount(maxPrice, price)
+
+  const containerClasses = classNames(
+    styles.skuSelectorItem,
+    `${styles.skuSelectorItem}--${slug(variationValue)}`,
+    'relative di pointer flex items-center outline-0',
+    {
+      [styles.skuSelectorItemImage]: isImage,
+      'o-20': isImpossible,
+    }
+  )
+
+  const passedAnyDimension = Boolean(imageHeight || imageWidth)
+  let containerStyles = {}
+  if (isImage && passedAnyDimension) {
```

Comment:
Shouldn't `imageUrl` be here as well? 

The way that it is right now, we could change the styles, but the if bellow could be `false`, making the styles changes unnecessary.